src/CommunityToolkit.Maui.Core/Services/DialogFragmentService.shared.cs

```diff
@@ -0,0 +1,14 @@
+using System.ComponentModel;
+using System.Runtime.Versioning;
+
+namespace CommunityToolkit.Maui.Core.Services;
+
+/// <summary>
+/// An Android-specific service to support <see cref="CommunityToolkit.Maui.Core.Platform.StatusBar"/> on modal pages
+/// </summary>
+[EditorBrowsable(EditorBrowsableState.Never)]
+[SupportedOSPlatform("Android")]
+public sealed partial class DialogFragmentService
```

Comment:
Just curious, why add a shared version if the feature is just for android?

Reply:
Good question! I added it when I was seeing if I could remove `#if ANDROID` from the shared code in this PR. 

I was ultimately unsuccessful in my attempt, but I decided to keep `DialogFragmentService.shared.cs` now that it's public since it could possibly help devs who want to reference it without an #IfDef:

```cs
if(OperatingSystem.IsAndroid())
{
  // Use `DialogFragmentService` without an #IfDef
} 
```